Description: Set dnsapi and deploy base folder as /usr/share/acme
 TODO: Put a short summary on the line above and replace this paragraph
 with a longer explanation of this change. Complete the meta-information
 with other relevant fields (see below for details). To make it easier, the
 information below has been extracted from the changelog. Adjust it or drop
 it.
 .
 acme (2.7.9-1~jessie+bearstech1) jessie-bearstech; urgency=medium
 .
   * Updated wrt. upstream' 2.7.9 branch (not yet released), fixes ACME v2 account creation
Author: Lucas BONNET <lbonnet@bearstech.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: https://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- acme-2.7.9.orig/acme.sh
+++ acme-2.7.9/acme.sh
@@ -11,6 +11,7 @@ PROJECT="https://github.com/Neilpang/$PR
 DEFAULT_INSTALL_HOME="$HOME/.$PROJECT_NAME"
 _SCRIPT_="$0"
 
+_SCRIPT_HOME="/usr/share/acme"
 _SUB_FOLDERS="dnsapi deploy"
 
 LETSENCRYPT_CA_V1="https://acme-v01.api.letsencrypt.org/directory"
